docs/v2/spec/2018_Modified_LAR_Code_Sheet.csv

```diff
@@ -0,0 +1,246 @@
+63,Debt-to-Income Ratio,20%-<30%,Debt-to-income ratio is between 20% and is less than 30%,
+63,Debt-to-Income Ratio,30%-<36%,Debt-to-income ratio is between 30% and is less than 36%,
+63,Debt-to-Income Ratio,50%-60%,Debt-to-income ratio is between 50% and 60%,
+63,Debt-to-Income Ratio,>60%,Debt-to-income ratio is greater than 60%,
+63,Debt-to-Income Ratio,NA,Not applicable,
+63,Debt-to-Income Ratio,Exempt ,Exemption taken ,
+67,Balloon Payment,1,Balloon payment,
+67,Balloon Payment,2,No balloon payment,
+67,Balloon Payment,1111,Exempt,
+68,Interest-Only Payments,1,Interest-only payments,
+68,Interest-Only Payments,2,No interest-only payments,
+68,Interest-Only Payments,1111,Exempt,
+69,Negative Amortization,1,Negative amortization,
+69,Negative Amortization,2,No negative amortization,
+69,Negative Amortization,1111,Exempt,
+70,Other Non-Amortizing Features,1,Other non-fully amortizing features,
+70,Other Non-Amortizing Features,2,No other non-fully amortizing features,
+70,Other Non-Amortizing Features,1111,Exempt,
+72,Manufactured Home Secured Property Type,1,Manufactured home and land,
+72,Manufactured Home Secured Property Type,2,Manufactured home and not land,
+72,Manufactured Home Secured Property Type,3,Not applicable ,
+72,Manufactured Home Secured Property Type,1111,Exempt,
+73,Manufactured Home Land Property Interest,1,Direct ownership,
+73,Manufactured Home Land Property Interest,2,Indirect ownership,
+73,Manufactured Home Land Property Interest,3,Paid leasehold,
+73,Manufactured Home Land Property Interest,4,Unpaid leasehold,
+73,Manufactured Home Land Property Interest,5,Not applicable ,
+73,Manufactured Home Land Property Interest,1111,Exempt,
+74,Total Units,1,1-Unit,
+74,Total Units,2,2-Unit,
+74,Total Units,3,3-Unit,
+74,Total Units,4,4-Unit,
+74,Total Units,24-May,Total units between 5 and 24,
+74,Total Units,25-49,Total units between 25 and 49,
+74,Total Units,50-99,Total units between 50 and 99,
+74,Total Units,100-149,Total units between 100 and 149,
+74,Total Units,>149,Total units over 149,
+76,Submission of Application,1,Submitted directly to your institution,
+76,Submission of Application,2,Not submitted directly to your institution,
+76,Submission of Application,3,Not applicable ,
+76,Submission of Application,1111,Exempt,
+77,Initially Payable to Your Institution,1,Initially payable to your institution,
+77,Initially Payable to Your Institution,2,Not initially payable to your institution,
+77,Initially Payable to Your Institution,3,Not applicable ,
+77,Initially Payable to Your Institution,1111,Exempt,
+78,Automated Underwriting System,1,Desktop Underwriter (DU),
+78,Automated Underwriting System,2,Loan Prospector (LP) or Loan Product Advisor,
+78,Automated Underwriting System,3,Technology Open to Approved Lenders (TOTAL) Scorecard,
+78,Automated Underwriting System,4,Guaranteed Underwriting System (GUS),
+78,Automated Underwriting System,5,Other. ,Note: Code 5 is only valid for Automated Underwriting System: 1

```

Comment:
The FIG has code 5 for each AUS. Can you verify which is correct and either change it here or let me know about the FIG?

Reply:
You are correct. Code 5 is available for each AUS. Thanks for the catch. This change is reflected in the last commit.